dubbo-plugin/dubbo-native/src/main/java/org/apache/dubbo/aot/generate/ReflectConfigMetadataRepository.java

```diff
@@ -78,6 +78,24 @@ private TypeDescriber buildTypeDescriberWithConstructor(Class<?> c) {
         return new TypeDescriber(c.getName(), null, new HashSet<>(), constructors, new HashSet<>(), memberCategories);
     }
 
+    protected ReflectConfigMetadataRepository registerFieldType(List<Class<?>> classes) {
+        types.addAll(classes.stream()
+                .filter(Objects::nonNull)
+                .map(this::buildTypeDescriberWithField)
+                .collect(Collectors.toList()));
+        return this;
+    }
+
+    private TypeDescriber buildTypeDescriberWithField(Class<?> c) {
+        Set<ExecutableDescriber> constructors = Arrays.stream(c.getConstructors())
+                .map((constructor) -> new ExecutableDescriber(constructor, INVOKE))
+                .collect(Collectors.toSet());
+        Set<MemberCategory> memberCategories = new HashSet<>();
+        memberCategories.add(MemberCategory.INVOKE_PUBLIC_METHODS);
```

Comment:
This type does not need to contain public method.

Reply:
Done.